### DISCUSSION TITLE
Stop Control for Shades/Blinds

### DISCUSSION THREAD

**Discussion by berryvew** [aarons22/homebridge-bond#265] [General]
Is there a way to support Stop control for shades or blinds? I am trying to set an automation that starts or closes them after x seconds. I have Open, Close, and Stop in Bond, but only Open or Close in HomeBridge with this plugin

**Comment by benderncb** [comment]
Also, the Bond app allows for ‘open next’ and ‘close next’. It would be nice to have those controls if possible.
